be-rust-axum/rust-template/chart/templates/_helpers.tpl

```diff
@@ -46,6 +46,7 @@ app.kubernetes.io/managed-by: {{ .Release.Service }}
 Selector labels
 */}}
 {{- define "chart.selectorLabels" -}}
+app: {{ .Values.componentId }}
```

Comment:
@serverhorror  this is definitely something to take note for the other helm implementations!
kudos to @braisvq1996! 🚀